.github/workflows/top-issues.yml

```
@@ -0,0 +1,21 @@
+name: Label Top Issues
+
+on:
+  schedule:
+  #- cron:  '15 0-23 * * *'
+  push:
+  #  branches: [top-issues]
```

I think cron make sense here, better than push.
Maybe once a week is engouh ?
```suggestion
  schedule:
  # every sunday at 23
  - cron:  '0 23 * * 0'
```